src/components/MenuComponent.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+
+import KarakterComponent from "./KarakterComponent";
+import KnowledgeComponent from "./KnowledgeComponent";
+import AchievementsComponent from "./AchievementsComponent";
+import QuestsComponent from "./QuestsComponent";
+import HistoryComponent from "./HistoryComponent";
+
+import styled from "styled-components";
+
+import { BrowserRouter as Router, Link, Route } from "react-router-dom";
+
+export default function MenuComponent() {
+  return (
+    <Router>
+      <Menu>
+        <ul>
+          <li>
+            <Link to="/karakter">
+              <i className="fa fa-female"></i>
+              <Tooltip>Karakter</Tooltip>
+            </Link>
+          </li>
+          <li>
+            <Link to="/knowledge">
+               <i className="fa fa-graduation-cap"></i>
+              <Tooltip>Knowledge Book</Tooltip>
+            </Link>
+          </li>
+          <li>
+            <Link to="/achievement">
+               <i className="fa fa-trophy"></i>
+              <Tooltip>Achievements</Tooltip>
+            </Link>
+          </li>
+          <li>
+            <Link to="/quests">
+               <i className="fa fa-question"></i>
+              <Tooltip>Quests</Tooltip>
+            </Link>
+          </li>
+          <li className="menu__last-item">
+            <Link to="/history">
+              <i className="fa fa-history"></i>
+              <Tooltip>History</Tooltip>
+            </Link>
+          </li>
+        </ul>
+      </Menu>
+      <Route path="/karakter" component={KarakterComponent} />
+      <Route path="/knowledge" component={KnowledgeComponent} />
+      <Route path="/achievement" component={AchievementsComponent} />
+      <Route path="/quests" component={QuestsComponent} />
+      <Route path="/history" component={HistoryComponent} />
+    </Router>
+  );
+}
+
+const Menu = styled.nav`
+  background: #f97970;
+  position: absolute;
```

Comment:
waarom is deze position absolute? Het lijkt erop alsof je ruimte reserveert voor het menu in je grid, (90px),  maar als je dan position absolute gebruikt, komt deze er niet in.

Ik zie overigens dat de positionering ook niet goed gaat ten opzicht van de rest van de pagina, ik zie een strook wit: 
![Screenshot 2021-03-15 at 12 47 30](https://user-images.githubusercontent.com/8133969/111149524-46d24380-858d-11eb-8be2-9bd05d09e443.png)

Reply:
dat kwam door de verkeerde vh, de 90px zijn inderdaad voor menu

Reply:
maar deze is nog steeds position absolute... ?